examples/InformationPanel.horizontal.tsx

```diff
@@ -14,11 +15,12 @@ import {
   Table,
   Text,
   InputGrid,
+  IconButton,
 } from '@itwin/itwinui-react';
-import * as React from 'react';
+import { SvgWindowPopout } from '@itwin/itwinui-icons-react';
 
 export default () => {
-  const [openRowIndex, setOpenRowIndex] = React.useState<number>(1);
+  const [openRowIndex, setOpenRowIndex] = React.useState<number>();
```

Comment:
Should `openRowIndex` start with `-1` to be consistent with the close button's logic that sets the index to -1 when the panel is closed?